tutorials/get-started/index.md

```diff
@@ -0,0 +1,989 @@
+import 'package:english_words/english_words.dart';
+
+void main() => runApp(new MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    var wordPair = new WordPair.random();
+
+    return new MaterialApp(
+      title: 'Welcome to Flutter',
+      home: new Scaffold(
+        appBar: new AppBar(
+          title: new Text('Welcome to Flutter'),
+        ),
+        body: new Center(
+          child: new Text(wordPair.asPascalCase),
+        ),
+      ),
+    );
+  }
+}
+{% endprettify %}
+
+---
+
+## Step 3: Add a Stateful widget
+
+State<em>less</em> widgets are immutable, meaning that their
+properties can’t change&mdash;all values are final.
+
+State<em>ful</em> widgets maintain state that might change
+during the lifetime of the widget. Implementing a stateful
+widget requires at least two classes: 1) a StatefulWidget class
+that creates an instance of 2) a State class. The StatefulWidget
+class is, itself, immutable, but the State class persists over the
+lifetime of the widget.
+
+In this step, you’ll add a stateful widget, RandomWords, which creates
+its State class, RandomWordsState. The State class will eventually
+maintain the proposed and favorite word pairs for the widget.
+
+<ol markdown="1">
+<li markdown="1">
+Add the stateful RandomWords widget to main.dart. It can go
+anywhere in the file, outside of MyApp, but the solution places
+it at the bottom of the file. The RandomWords widget does little
+else besides creating its State class:
+
+<!-- skip -->
+{% prettify dart %}
+class RandomWords extends StatefulWidget {
+  @override
+  createState() => new RandomWordsState();
+}
+{% endprettify %}
+</li>
+
+<li markdown="1">
+Add the RandomWordsState class. Most of the
+app’s code resides in this class, which maintains the state for the
+RandomWords widget. This class will save the generated word pairs,
+which grow infinitely as the user scrolls, and also favorite
+word pairs, as the user adds or removes them from the list by
+toggling the heart icon.
+
+You’ll build this class bit by bit. To begin, create a minimal
+class by adding the highlighted text:
+
+<!-- skip -->
+{% prettify dart %}
+class RandomWordsState extends State<RandomWords> {
+}
+{% endprettify %}
+</li>
+
+<li markdown="1">
+After adding the state class, the IDE complains that the class is
+missing a `build` function. Next, you'll add a basic
+build function that generates the word pairs by moving the
+word generation code from MyApp to RandomWordsState.
+
+Add the `build` function to RandomWordState, as shown
+by the highlighted text:
+
+<!-- skip -->
+{% prettify dart %}
+class RandomWordsState extends State<RandomWords> {
+  [[highlight]]@override[[/highlight]]
+  [[highlight]]Widget build(BuildContext context) {[[/highlight]]
+    [[highlight]]var wordPair = new WordPair.random();[[/highlight]]
+    [[highlight]]return(new Text(wordPair.asPascalCase));[[/highlight]]
+  }
```

Comment:
This closing curly brace should be highlighted, too, since it's also new. I didn't copy this and the code won't compile.

Reply:
Thanks for catching that! Done